docs/user_guide/capsules/index.md

```diff
@@ -1,43 +1,53 @@
 layout: documentation
-title: Capsules 
+title: Capsules
 sidebar: sidebars/documentation.html
 ---
 
 # Capsules
 
 ## What are Capsules?
- The Capsule Server is a separate server that provides services to discover, provision, and configure hosts remotely. The Capsule provides the following features:
-        
-  * Content delivery (Mirror lifecycle enivronments from your Katello server)
-  * DHCP, including ISC DHCP
-  * DNS, including Bind and MS DNS servers
-  * Any UNIX-based TFTP server
-  * Puppet Master servers from 0.24
-  * Puppet CA to manage certificate signing and cleaning
-  * Baseboard Management Controller (BMC) for power management 
 
-Not all of these features need to be deployed onto a Capsule, as they are all optional.  For example, if in a particular network segment dhcp and dns are managed already (possibly by another group) and you only want the content mirroring capability, you can deploy a Capsule with just the content delivery portion.  
-
-The end goal is for all client communication to be able to go through a Capsule server.  Isolating communication through the Capsule prevents the client from needing network connectivity to the Katello server.  There are still a few cases where clients need to communicate directly to the Katello server, but we hope to close those gaps in future releases. 
-
-With a fully configured Capsule, Hosts would boot and get their IP information from the Capsule's DHCP server, PXE boot from the Capsule's tftp server, resolve hostnames using the Capsule's DNS server, provision from kickstart trees that have been synced to the Capsule, configure itself using the Puppet Master running on the Capsule, and recieve content updates directly from the Capsule itself.
+A Capsule provides services to discover, provision, control, and configure hosts.  Katello servers include what's known as the default capsule, and you may deploy additional capsules to various different network segments to manage servers there.  The Capsule provides the following features:
+
+  * Content delivery
+  * Consumer action delivery (package installation, etc)
+  * Proxy for subscription-manager
+  * Traditional Foreman Proxy features:
+    * DHCP, including ISC DHCP
+    * DNS, including Bind and MS DNS servers
+    * Any UNIX-based TFTP server
+    * Manage a Puppetmaster
+    * Puppet CA to manage certificate signing and cleaning
+    * Baseboard Management Controller (BMC) for power management
+    * Realm (FreeIPA)
+    * Provisioning templates
+
+Not all of these features need to be deployed onto a Capsule, as many are optional.  For example, if in a particular network segment DHCP and DNS are managed already, you may only want the Content delivery capability, you can deploy a Capsule with just that feature.
+
+However, the goal if capsules is to have an endpoint through which all client communication can be routed. See [Capsule Isolation](/docs/{{ site.version }}/user_guide/capsules/isolation.html) for more information.
```

Comment:
This sentence reads weird to me - I think its trying to tell me that the goal of capsules is as a client communication router?